packages/CollapsibleChecklists/src/components/Heading/Heading.styles.js

```diff
@@ -0,0 +1,11 @@
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+
+const headingStyles = () => `
```

Comment:
better use `css`\` in these styles? https://github.com/acl-services/paprika/issues/98